rock_gazebo.orogen

```diff
@@ -49,10 +49,17 @@ task_context "ModelTask" do
     # The covariance matrix to be used as velocity covariance. Default is unset (NaN)
     property 'cov_velocity', '/base/Matrix3d'
 
+    # The acceleration of the model w.r.t. the world frame
+    output_port 'ang_accel_samples', '/base/samples/RigidBodyAcceleration'
+    # The covariance matrix to be used as acceleration covariance. Default is unset (NaN)
+    property 'cov_acceleration', '/base/Matrix3d'
+
     property "exported_links", "std/vector<rock_gazebo/LinkExport>"
 
     # The model's pose if one wants to "warp"
     input_port 'model_pose', "/base/samples/RigidBodyState"
+    # Apply wrench to the model
+    input_port 'wrenches', "/base/samples/Wrenches"
```

Comment:
Our use case is to apply the wrench in only one link, the vessel, to control its position and orientation during simulation. I was thinking of implementing this using the exported_links, but I couldn't get it work, so I used the samples/Wrenches instead.